docs/sdk/java.md

```diff
@@ -162,7 +162,7 @@ Finally, if a catch-all exists, it will be executed or an ``IllegalStateExceptio
 
 ## Function Types and Messaging
 
-In Java, function types are defined as a _stringly_ typed reference containing a namespace and name.
+In Java, function types are defined as a _strongly_ typed reference containing a namespace and name.
```

Comment:
It is `stringly typed` right @sjwiesman ?

Reply:
I used stringly on purpose because you aren't refereing to the class but a string "<namespace>/<name>". I don't think strongly is correct but we can change it. How about, 

```
In Java, function types are defined as logical pointers composed of a namespace and name
```
?

Reply:
Yes, my suggestion didn't make sense. +1 to change it as Seth outlined.

"stringly typed" has a negative connotation for me as in [needlessly using strings when other options are available](https://wiki.c2.com/?StringlyTyped).

Reply:
Applying Seth's suggestion.